docs/root/configuration/observability/access_log/usage.rst

```diff
@@ -1111,6 +1111,24 @@ UDP
   UDP
     Not implemented ("-").
 
+%DOWNSTREAM_PEER_CHAIN_FINGERPRINTS_256%
+  HTTP/TCP/THRIFT
+    The hex-encoded SHA256 fingerprints of all client certificates used to establish the downstream TLS connection.
```

Comment:
Please specify/document how they are combined. Is this a comma-separated list? Are they all hashed together?